src/actions/users.js

```diff
@@ -1554,6 +1554,45 @@ export function clearUserAccessTokens() {
     };
 }
 
+// Send a getUserAnalytics request for a user (userId) and team id (teamId)
+// Pass a string indicating which datasets to return (name)
```

Comment:
s/b dataset (singular since the change to name instead of an array of names)

Reply:
Fixed.